apps/site/src/components/NavBar/NavBar.module.scss

```diff
@@ -3,73 +3,58 @@
 
 .nav {
 	width: 100%;
-	filter: drop-shadow(0px 2.5px 2px rgba(0, 0, 0, 0.4));
+	z-index: 101;
 
-	@include utils.media-breakpoint-up(sm) {
-		position: sticky;
-		z-index: 101;
-		top: 0;
+	@include utils.media-breakpoint-up(lg) {
+		background-color: transparent;
+		color: white;
+		backdrop-filter: none;
 	}
+
+	background-color: rgba(20, 20, 20, 0.7);
+	position: fixed;
+	text-align: right;
+	backdrop-filter: blur(7px);
 }
 
 .navbar {
-	background-image: url("~@/assets/images/lined_paper.svg");
-	background-size: cover;
-	background-position: left bottom;
-	background-repeat: no-repeat;
+	position: absolute;
+	top: 0;
 }
 
 .logo {
-	position: absolute;
-	left: 30px;
-	width: 20vw;
-	height: 20vw;
-	text-align: center;
-	transform: rotate(-16.81deg) translateY(-5vw);
+	position: relative;
+	width: 50px;
+	height: 50px;
+	align-items: left;
+	text-align: left;
+	top: 0;
 }
 
 .text {
 	margin-left: 1vw;
 	margin-right: 1vw;
-	font-size: 2rem;
-	font-weight: bold;
+	margin-top: 2vw;
+	margin-bottom: 2vw;
```

Comment:
Thought: by removing this, the links on the right of the navbar will be aligned horizontally with the ZotHacks logo. To me it looks slightly better. Do you think this will look worse than what the navbar looks right now? You can also push the links slightly left and maybe increase the font size.